test/middleware/navigation/has.trust.middleware.spec.ts

```diff
@@ -0,0 +1,89 @@
+jest.mock(".../../../src/utils/application.data");
```

Comment:
can you add the ioredis mock as first line and add blank line between jest.mocks and imports

Reply:
Added

Reply:
i can't seem to see the changes for this, are you sure the changes were pushed to github

Reply:
my bad, should be up now